index.html

```diff
@@ -477,8 +477,20 @@ <h4>description</h4>
           </section>
           <section class="normative">
             <h3>Representations of JSON Schema</h3>
-            <p class="issue" data-number="142">
-               Write section on using different representations of JSON Schema, such as YAML.
+            <p>
+               The standard representation of [[JSON-SCHEMA]] uses the [[RFC8259]] JSON data interchange
+               syntax</a> with <code>.json</code> as the file extension.
+            </p>
+            <p>
+               Implementers MAY choose to use OpenAPI Specification's [[OAS]] [[YAML]] representation
```

Comment:
I think this is important for interoperability.
```suggestion
               Implementers MUST choose to use OpenAPI Specification's [[OAS]] [[YAML]] representation
```

Reply:
I want to convey that if they MAY use YAML 
we should not require implementers to use YAML

Reply:
If they `MUST`, they cannot `choose`. I suggest this `MUST choose to use` be changed to either `MUST use` or `MAY use`.

I also suggest there be addition of something that indicates that some lossless data format transformation (i.e., between JSON and YAML) will be necessary for some interop.

Reply:
thanks, updated to `MAY use`